lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -29,6 +29,31 @@
     "disableStorageReset": false,
     "emulatedFormFactor": "mobile",
     "channel": "cli",
+    "budgetsJSON": [
+    {
```

Comment:
nit: would you mind cleaning up the indentation here? 

just to make the diffs cleaner for the next person who comes along to auto-update this :)

(we just use `JSON.stringify(json, null, 2)`)